webapi/src/main/scala/org/knora/webapi/routing/RouteUtilZ.scala

```diff
@@ -39,6 +39,35 @@ object RouteUtilZ {
         )
       )
 
+  def ensureExternalOntologyName(iri: SmartIri): ZIO[StringFormatter, BadRequestException, SmartIri] =
+    ZIO.serviceWithZIO[StringFormatter] { sf =>
```

Comment:
is this the ontology name or IRI? reads more like IRI, so it would seem the method name is misleading

Reply:
The original check was based on the `checkExternalOntologyName` of the `StringFormatter` which checks that the iri is a `SmartIri.isKnoraApiV2DefinitionIri` and that the `SmartIri.getOntologyName` is one of `knora-base` or `knora-admin`.

So probably the newly extracted method `org.knora.webapi.messages.StringFormatter#isKnoraOntologyIri` which contains above check should have a better name, e.g. `isKnoraOntologyName`?